extension/oteleventually/eventstore.go

```diff
@@ -117,9 +119,12 @@ func (es *InstrumentedEventStore) Stream(
 	case stream.ByTypes:
 		attributes = append(attributes, StreamTypeAttribute.StringSlice(t))
 	case stream.ByID:
-		attributes = append(attributes, StreamTypeAttribute.String(t.Type), StreamNameAttribute.String(t.Name))
+		attributes = append(attributes, StreamTypeAttribute.String(t.Type))
+		spanAttributes = append(spanAttributes, StreamNameAttribute.String(t.Name))
 	}
 
+	spanAttributes = append(spanAttributes, attributes...)
```

Comment:
Oh, approved too quickly 🤦🏻‍♂️ 

You need to use this variable in the Span recorder call down here 👍🏻

Reply:
My mistake, still half asleep and didn't notice, but that was this commit https://github.com/get-eventually/go-eventually/commit/0c4c102445b1fcd4dc3ef95738bed27fad41b5b4